src/main/webapp/assets/modules/pie-chart/pie-chart.fac.js

```diff
@@ -71,9 +71,8 @@ angular.module('Pie').factory('pieFactory', function (utils, jsonValue, termServ
           }
         },
         tooltip: {
-          formatter: function() {
-            return sprintf("<b>%(salRange)s</b> <br/>a month in average for jobs in <b>%(label)s</b>",
-              terms[data4PieChart.labels.indexOf(this.key)]);
+          formatter: function () {
+            return sprintf("<b>%(salRange)s</b> <br/>a month in average for jobs in <b>%(label)s</b>", terms[this.point.index]);
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'sprintf' is not defined.